addons/point_of_sale/upgrades/1.0.2/post-deduplicate-uuids.py

```diff
@@ -32,7 +32,7 @@ def deduplicate_uuids(table):
             ids = [r[0] for r in cr.fetchmany(10000)]
             cr.execute(
                 f"UPDATE {table} SET uuid = (%s::json)->>(id::text) WHERE id IN %s",
-                [Json({id_: str(uuid.uuid4()) for id_ in ids}), ids]
+                tuple([Json({id_: str(uuid.uuid4()) for id_ in ids}), ids])
```

Comment:
I guess you meant this?
```suggestion
                [Json({id_: str(uuid.uuid4()) for id_ in ids}), tuple(ids)]
```